suite-common/wallet-utils/src/transactionUtils.ts

```diff
@@ -920,6 +920,24 @@ export const simpleSearchTransactions = (
     });
     txsToSearch.push(...foundTxsForAddress);
 
```

Comment:
Tip

`Tip: You can search for transaction IDs, addresses, labels, amounts and dates.` 

not updated